run-vmtest/merge_test_lists.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
```

Comment:
maybe add a comment explaining what the script does? it does a bit more than just merging lists.